cotracker/README.md

```diff
@@ -0,0 +1,74 @@
+---
+Paper: https://arxiv.org/pdf/2307.07635.pdf
+Official Website: https://co-tracker.github.io/
+Demo: https://huggingface.co/spaces/facebook/cotracker
+---
+# CoTracker
+
+A new state of the art architecture for optical flow and particle tracking proposed by meta. The key innovation in this project is the use of Transformer Architecture and the idea that tracking multiple particles concurrently, can help boos overall accuracy.
+
+https://github.com/velocitatem/PoW/assets/60182044/527a3377-4f59-435f-a0e1-b8b65a838b08
+
+
+## The Problem with Existing Methods
+
+Traditionally, there are two main ways to predict motion in videos:
+
+1. **Optical Flow**: This method estimates the motion of every single point in a video frame but only for a very short period.
+2. **Point Tracking**: This focuses on tracking specific points over an extended period but does it independently, without considering the relationship between points.
+
+Both methods have their limitations. For example, they struggle with tracking points that get temporarily hidden (occluded) or are part of the same object.
+
+## High Level Overview
+
+CoTracker is built on the idea that points in a video are often correlated. For example, if you're tracking the wheels of a car, it's likely that if one wheel moves, the other will too. The authors argue that by ignoring these correlations, we're missing out on improving the accuracy of tracking. So, they propose a new architecture that tracks multiple points jointly throughout an entire video sequence. This is a game-changer because it allows the model be more informed of the correlations between points that are being tracked.
+
+```mermaid
```

Comment:
If you are going into how the architecture of Cotracker works. Perhaps also include figures 2 and 3 for better understanding.